texttospeech/README.rst

```diff
@@ -9,7 +9,7 @@ models.
 
 .. _Alpha: https://github.com/GoogleCloudPlatform/google-cloud-python/blob/master/README.rst
 .. _Cloud Text-to-Speech API: https://cloud.google.com/texttospeech
-.. _Client Library Documentation: https://google-cloud-python.readthedocs.io/en/latest/texttospeech/index.html
+.. _Client Library Documentation: https://googlecloudplatform.github.io/google-cloud-python/stable/texttospeech/usage.html
```

Comment:
404s.  We aren't publishing `stable` docs at all.  Also, `index.html` is vaild, `usage.html` is not.